File: Level-2/20.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] arr = s.split(" ");
        for(int i = 0; i < arr.length; i++){
            for(int j = 0; j < arr[i].length(); j++){
                if(j == 0){
                    answer += String.valueOf(arr[i].charAt(j)).toUpperCase();
                }else{
                    answer += String.valueOf(arr[i].charAt(j)).toLowerCase();
                }
            }
            if(i != arr.length - 1) answer += " ";
        }
        if(s.charAt(s.length() - 1) == ' ') answer += " ";
        return answer;
    }
}

File: Level-2/20.js
function solution(arr) {
    let answer = 1;
    
    const gcd = (a, b) => {
        if(b === 0) return a;
        return gcd(b, a % b);
    }
    
    const lcm = (a, b) => {
        return a * b / gcd(a, b);
    }
    
    for(let i = 0; i < arr.length; i++) {
        answer = lcm(answer, arr[i]);
    }
    
    return answer;
}
